packages/flame/lib/src/components/router_component.dart

```diff
@@ -104,6 +104,34 @@ class RouterComponent extends Component {
     _adjustRoutesVisibility();
   }
 
+  /// Pops the current route and places the route [name] on top of the
+  /// navigation stack.
+  ///
+  /// If the route is already in the stack, it will be simply moved to the top.
+  /// Otherwise the route will be mounted and added at the top. We will also
+  /// initiate building the route's page if it hasn't been built before. If the
+  /// route is already on top of the stack, this method will do nothing.
```

Comment:
```suggestion
  /// Otherwise the route will be mounted and added at the top. The route's
  /// page will also start building if it hasn't been built before. If the
  /// route is already on top of the stack, this method will do nothing.
```
Avoid "we" in dartdocs

Reply:
Thats what you guys wrote, lol!  Love that git blame.